api-reference/10 UI Components/dxForm/1 Configuration/showColonAfterLabel.md

```diff
@@ -5,7 +5,7 @@ default: true, false (Material)
 ---
 ---
 ##### shortDescription
-Specifies whether or not a colon is displayed at the end of form labels.
+Specifies whether or not a colon is displayed at the end of form labels. Applies only if the [labelMode](/api-reference/10%20UI%20Components/dxForm/1%20Configuration/labelMode.md '/Documentation/ApiReference/UI_Components/dxForm/Configuration/#labelMode') is *"outside"* or [visible](api-reference\10%20UI%20Components\dxForm\5%20Item%20Types\SimpleItem\label\visible.md '/Documentation/ApiReference/UI_Components/dxForm/Item_Types/SimpleItem/label/#visible') is set to a true for the labels.
```

Comment:
In the editor's labels, the option will have no effect and the colon will not be displayed